### ISSUE TITLE
Example links in README result in 404s

### ISSUE THREAD

**Issue by skwashd** [awslabs/aws-lambda-powertools-python#184]
**What were you trying to accomplish?**

Review the example project

## Expected Behavior

When I click the example links I should see the example project

## Current Behavior

Clicking the link results in a 404

## Possible Solution

This was introduced by #180. Either reinstate the example project or remove the reference to it in the docs.

## Steps to Reproduce (for bugs)

1. Navigate to https://github.com/awslabs/aws-lambda-powertools-python
2. Click the kitchen sink example link
3. See a 404 page
4. Wonder if you've done something wrong
5. Swear
6. Click the browser's back button
7. Find the example link further down the page
8. Click the link
9. Get another 404
10. Click back
11. Swear

## Environment

* **Powertools version used**: develop @ d08de0b
* **Packaging format (Layers, PyPi)**: GitHub web UI
* **AWS Lambda function runtime:** N/A
* **Debugging logs** N/A

![Screen Shot 2020-09-30 at 11 36 05 am](https://user-images.githubusercontent.com/45939/94633617-385edb00-0311-11eb-84ae-43d848870336.png)

**Comment by heitorlessa** [comment]
Hi @skwashd - Thanks for spotting this.

I've recently removed to unblock another PR and accidentally didn't update the README links - Use this below instead

https://github.com/aws-samples/cookiecutter-aws-sam-python

If using SAM CLI, you can instantiate using: sam init --location https://github.com/aws-samples/cookiecutter-aws-sam-python

And these are more concrete examples:

https://github.com/aws-samples/aws-serverless-shopping-cart

https://github.com/aws-samples/aws-serverless-ecommerce-platform

https://github.com/aws-samples/aws-serverless-airline-booking

**Comment by skwashd** [comment]
Thanks for the links.
